src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationBlockCommentBeforePackage.java

```diff
@@ -0,0 +1,10 @@
+/* // violation
```

Comment:
how this input related to this issue?

Reply:
This is to cover cases of comments when there is no parent node of Javadoc. 
https://github.com/checkstyle/checkstyle/blob/6c2851302355d082d809775b0ff7b9c704132c1a/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheck.java#L1195-L1197